docs/fundamentals/executor/containerize-executor.md

```diff
@@ -0,0 +1,155 @@
+# Dockerize your Executor
+
+Once you have understood what an `Executor` is and how it can be used inside a `Flow`, you may be interested om how to wrap this Executor into a container
```

Comment:
```suggestion
Once you have understood what an `Executor` is and how it can be used inside a `Flow`, you may be interested on how to wrap this Executor into a container
```